### ISSUE TITLE
Preview Button

### ISSUE THREAD

**Issue by tashsmit** [RosmaryFC/Meme-ify-Me-ARHM#39]
The preview button must be clicked in order to update the picture so it is saved properly. Maybe it would be better to make sure the preview buttons are clicked at least once before the save button is enabled; that way the user understands the picture must be previewed before it is updated and saved.

**Comment by abassawo** [comment]
Good point, Tasha. And I think that could be easily handled with a toast msg.
